app/serializers/logs_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class LogsSerializer < ApplicationSerializer
+  attributes :id, :item_type, :event, :user_id, :username, :object, :created_at
+
+  belongs_to :encryptable
```

Comment:
make sure to use `include` where you are fetching the version entries to avoid n+1 queries.